src/main/resources/config.sk

```diff
@@ -31,7 +31,7 @@
 # ==== General Options ====
 
 language: english
-# Which language to use. Currently english, german and korean are included in the download, but custom languages can be created as well.
+# Which language to use. Currently english, german korean and polish are included in the download, but custom languages can be created as well.
```

Comment:
You forgot a comma between german and korean

`german, korean`

Reply:
Fixed thanks